packages/core/src/tests/input/dynamic/index.html

```diff
@@ -0,0 +1,46 @@
+<!--
+SPDX-FileCopyrightText: 2024 Siemens AG
```

Comment:
```suggestion
SPDX-FileCopyrightText: 2025 Siemens AG
```

File was created in 2025.